src/components/providers/chakra/provider.tsx

```
@@ -1,17 +1,56 @@
+    const names: string[] = [];
+    let styles = '';
+    for (const [name, value] of entries) {
+      if (typeof value === 'string') {
+        names.push(name);
+        styles += value;
+      }
+    }
+
+    return (
+      <style
+        dangerouslySetInnerHTML={{ __html: styles }}
```

Biome (2.3.14)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)